puppet/modules/slave/manifests/unittests.pp

```diff
@@ -167,5 +167,12 @@
   include slave::postgresql
 
   # RVM
-  include slave::rvm
+  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] != '9' {
```

Comment:
This means Debian nodes don't get neither rvm nor rbenv, which is fine as we do not run unittests there, but needs tests fixed